app/views/RoomView/index.js

```diff
@@ -72,6 +71,14 @@ class RoomView extends React.Component {
 					navigation={navigation}
 					toggleFollowThread={toggleFollowThread}
 				/>
+			),
+			headerLeft: (
+				<HeaderBackButton
+					title={unreadsCount > 999 ? '+999' : unreadsCount}
+					backTitleVisible
+					onPress={navigation.goBack}
```

Comment:
I don't understand why, but this not work on Android.. only `() => navigation.goBack()`